programs/models.py

```diff
@@ -186,13 +186,19 @@ def create_instance(cls, external_name: str, Model: type["Document"]) -> "Docume
 class Document(models.Model):
     external_name = models.CharField(max_length=120, blank=True, null=True, unique=True)
     text = models.ForeignKey(Translation, related_name="documents", blank=False, null=False, on_delete=models.PROTECT)
+    link_url = models.ForeignKey(
+        Translation, related_name="document_link_url", blank=True, null=True, on_delete=models.PROTECT
+    )
+    link_text = models.ForeignKey(
+        Translation, related_name="document_link_text", blank=True, null=True, on_delete=models.PROTECT
+    )
```

Comment:
We use empty translations for empty fields instead of `null` because it makes it so that we don't have to recreate the translation object if we want to create a link.